docs/topics/coroutines-basics.md

```diff
@@ -250,14 +250,14 @@ Done
 Coroutines are less resource-intensive than JVM threads. Code that exhausts the
 JVM's available memory when using threads can be expressed using coroutines
 without hitting resource limits. For example, the following code launches
-100000 distinct coroutines that each wait 5 seconds and then print a period
+10000 distinct coroutines that each wait 5 seconds and then print a period
 ('.') while consuming very little memory:
 
 ```kotlin
 import kotlinx.coroutines.*
 
 fun main() = runBlocking {
-    repeat(100_000) { // launch a lot of coroutines
+    repeat(10_000) { // launch a lot of coroutines
```

Comment:
This example is non-runnable, so there's no issue with `100_000` here. In this case, we intentionally highlight the difference from threads, and it's more important to have a whooping 100_000 coroutines here than to be able to use https://play.kotlinlang.org/.

Reply:
Sorry, I'm not sure I agree here. Should we at least change the wording, or warn that the playground doesn't handle this? 

The reason I filed the bug [KT-51515](https://youtrack.jetbrains.com/issue/KT-51515) over a year ago is because I was amazed when I saw this example, and immediately popped into the playground to see it in action. But of course it crashed and it leaves a bad impression about trusting the documentation.

Reply:
> Should we at least change the wording, or warn that the playground doesn't handle this?

I agree, we should at least add a warning. @danil-pavlov, could you help us with that?